mis_builder/static/src/components/mis_report_widget.esm.js

```diff
@@ -138,6 +142,21 @@ export class MisReportWidget extends Component {
             [this._instanceId()],
             {context: this.context}
         );
+        this.state.can_edit_annotation = await this.user.hasGroup(
+            "mis_builder.group_edit_annotation"
+        );
+        this.state.can_read_annotation = await this.user.hasGroup(
+            "mis_builder.group_read_annotation"
+        );
```

Comment:
Let's add fields on mis.report.instance so the logic to decide if annotations can be read or written is easier to override.